microk8s-resources/wrappers/microk8s-kubectl.wrapper

```diff
@@ -17,6 +17,5 @@ if echo "$*" | grep -v -q -- '--kubeconfig'; then
   exit_if_no_permissions
 fi
 
-
-declare -a args="($(cat $SNAP_DATA/args/kubectl))"
-"${SNAP}/kubectl" "${args[@]}" "$@"
+export KUBECONFIG=$SNAP_DATA/credentials/client.config
```

Comment:
I am not sure about this. In the past we had users complaining that they could not change the location of the kubeconfig file so we added this arguments file.

What if we had a kubectl-env file with `export KUBECONFIG=$SNAP_DATA/credentials/client.config` and we sourced that file instead of hardcoding the client.config location here? We could leave the args/kubectl file empty incase someone needs it. What do you think?

Reply:
Oh yes. I didn't consider that users can change the content of the config.
Sounds fair.